broid-slack/README.md

```diff
@@ -40,15 +40,42 @@ You can generate one for test with this command: ``node bin/oauth.js --new -c cl
 const BroidSlack = require('broid-slack');
 
 const slack = new BroidSlack({
+  token: 'xxxxx',
+  http: {
+    host: '127.0.0.1',
+    port: 8080
+  }
+});
+
+slack.connect()
+  .subscribe({
+    next: data => console.log(data),
+    error: err => console.error(`Something went wrong: ${err.message}`),
+    complete: () => console.log('complete'),
+  });
+```
+
+Twilio can also be used with your existing express setup.
+
+```javascript
+const BroidSlack = require('broid-slack');
+const express = require("express");
```

Comment:
We could for sure, but it's nice to have a quick easy example right in the module readme as well.